<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeaponDesigner</class>
 <widget class="QDialog" name="WeaponDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>370</width>
     <height>30</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>One Handed Sword</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/ICO/oneHadedSword.png</normaloff>:/ICO/oneHadedSword.png</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Two Handed Sword</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>One Handed Axe</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>twoHandedAxe</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>club</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>polearm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>spear</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dagger</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>knife</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>One Handed Hammer</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Two Handed Hammer</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>291</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Materials</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Build">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>410</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>211</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Build price:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Quality:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Sell price:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_BuildPrice">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Quality">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_SellPrice">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Evaluate">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>410</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Evaluate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
